source/user/mcscf.rst

```diff
@@ -1,57 +1,299 @@
-.. _theory_mcscf:
+.. _user_mcscf:
 
 Multi-configuration self-consistent field (MCSCF)
 *************************************************
 
 *Modules*: :mod:`mcscf`
 
 
+Introduction
+------------
 
-Symmetry in CASSCF
-------------------
+Multiconfigurational self-consistent field (MCSCF) methods extend Hartree-Fock (HF) theory by describing the wave function as a linear combination of determinants. 
+Unlike full configuration interaction (FCI), described in :numref:`theory_ci`, complete active space (CAS) methods perform the FCI procedure on a subset of the molecular orbitals, referred to as the "active space." 
+These methods are crucial for systems that exhibit strong electron correlation, such as transition metal complexes.
+For a detailed discussion of MCSCF methods, we direct the reader to :cite:`Helgaker2013` and :cite:`esqc`.
 
-.. literalinclude:: /../examples/mcscf/21-nosymhf_then_symcasscf.py
+The MCSCF module has two main flavors: CASCI and CASSCF. 
+CASCI uses the single reference orbitals from a HF calculation and solves the CI eigenvalue problem once.
+CASSCF calculations optimize the orbitals for the set of determinants and require solving the CI eigenvalue multiple (and possibly many) times.
```

Comment:
The reference orbitals for CASCI may not be HF, you can also use other references like DFT, MP2 natural orbitals, etc.

Maybe it would be clearer to say that a CASSCF calculation consists of multiple CASCI steps, where the wave function computed at each step gives you an orbital gradient that is used to optimize the active space.

Reply:
Strictly speaking, I think CASCI orbitals should be HF, but practically speaking I know we can use other orbitals. I will change this tomorrow. I'll also remove the eigenvalue problem language as you suggested.

Reply:
Yes. Common examples are CASCI(MP2) or CASCI(CISD). Using natural orbitals from MP2 or CI results in much faster convergence of the CI expansion than HF virtuals.